tenable/io/v3/vm/permissions/api.py

```diff
@@ -44,13 +44,13 @@ def change(self, otype: str, id: UUID, *acls: Dict) -> None:
                 `permissions documentation`_ for more details.
```

Comment:
Is the `_` here intended?

Reply:
yes, it is the syntax required for adding links in documentation.